### ISSUE TITLE
[Feat. request] Show list of open buffers in UI

### ISSUE THREAD

**Issue by GitSquared** [jmacdonald/amp#141]
It would be nice to have some kind of tab bar or side list to see what buffers are currently open, and maybe number-based keyboard shortcuts to switch to them instead of hitting `tab` multiple times.

It seems to me that having the ability to open multiple buffers without being able to see and efficiently switch between them is like a half-implemented tabs feature - and that kind of contradicts with the decision to delegate split panes support to the user's terminal emulator. What is your take on this?

(Possibly related: #89 #133)

**Comment by jmacdonald** [comment]
Hi @GitSquared!

It's definitely on the list of things to tackle, and I've got various ideas around how I'd like to implement it. I'm going to tag this issue so I can revisit it once 0.6 lands.

**Comment by BearOve** [comment]
I added PR's to scribe and amp with a version of this I have been using in my own fork since August in the hope that it helps out this issue.

**Comment by cecton** [comment]
(This is the only feature I'm waiting before trying amp again.)
